src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1275,4 +1275,16 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="SystemArchitecture" xml:space="preserve">
     <value>System Architecture</value>
   </data>
+  <data name="PreserveArgumentDescription" xml:space="preserve">
+    <value>Retains all files and directories created by the package (portable)</value>
+  </data>
+  <data name="PurgeArgumentDescription" xml:space="preserve">
+    <value>Deletes all files and directories created by the package (portable)</value>

```

Comment:
```suggestion
    <value>Deletes all files and directories in the package directory (portable)</value>
```
We cannot delete them if they were created outside of the package directory (and I don't think we should delete them unless we created the directory either).

Reply:
Changed to suggested